### ISSUE TITLE
Upgrading to 0.4.4 causes the month I scroll to, to be blank

### ISSUE THREAD

**Issue by ganesh3s3** [kizitonwose/CalendarView#218]
Everything works perfectly fine when I revert back to 0.4.3 but with 0.4.4, the month that I scroll to is blank initially and only becomes visible when I scroll to the other visible months.

I setup the calendarview 

`exTwoCalendar?.setup(
            YearMonth.of(2019, 1),
            YearMonth.now().plusYears(1),
            daysOfWeek.first()
        )`

and immediately scroll to the current month

`exTwoCalendar?.scrollToMonth(YearMonth.now())`

Here is my CalendarView:

`<com.kizitonwose.calendarview.CalendarView
                android:id="@+id/exTwoCalendar"
                android:layout_width="0dp"
                android:layout_height="wrap_content"
                app:cv_dayViewResource="@layout/calendar_day"
                app:cv_monthHeaderResource="@layout/calendar_header"
                app:cv_orientation="vertical"
                app:cv_scrollMode="paged"
                app:layout_constraintBottom_toBottomOf="parent"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toBottomOf="@+id/legendLayout" />`

Please let me know if you need anything else.

**Comment by kizitonwose** [comment]
Can you build and run the sample project on your device and see if this issue happens with it? 

Note: Don't download and install the apk available on Github for this test as it was compiled against a previous version of the library.

**Comment by ganesh3s3** [comment]
Hey, sorry for the late reply. I built and ran the sample app like you said and I faced no issues.

BTW The calenderView xml I posted above is inside a ConstraintLayout which itself is inside a MotionLayout so I'm not sure if that is causing the issues between library versions. 

I haven't had much time to test out what the exact cause of this issue is so I made 2 videos of the exact same app except one uses CalenderView library version 0.4.3 and the other 0.4.4, to better showcase the problem I'm facing. 

**CalenderView 0.4.3:**
![CalenderView-0 4](https://user-images.githubusercontent.com/13646665/94415293-0592d680-019b-11eb-8266-3b3f66b9b029.gif)



**CalenderView 0.4.4:**

![0 4 4](https://user-images.githubusercontent.com/13646665/94423214-e77ea380-01a5-11eb-9e69-5898369169b5.gif)
